docs/Tutorial_Advanced_Annotation_Vectors.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>I have uploaded the file to Zenodo: https://zenodo.org/record/5067801/files/finger_flexion.csv?download=1</p><p> </p><p>Note that the column name has been changed to "Finger Flexion" (instead of "Finger Flexion Data")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>